### PR TITLE
updete TalkDetailView

### PR DESCRIPTION
**Pull request by Maliaotw**
## Types of changes

- [x] **New feature**

## Description

1. add TalkDetail Response to API。
2. update 'api' in path not use flat_page method.

## Steps to reproduce the behavior:

    Go to 'http://127.0.0.1:8000/api/talk/<int:pk>'

### CODE REVIEW

File: src/events/serializers.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            "title",
+            "category",
+            "language",
+            "python_level",
+            "recording_policy",
+            "abstract",
+            "detailed_description",
+            "slide_link",
+            "slido_embed_link",
+            # "sponsored"
```
Comment (hsuanchi) [inline src/events/serializers.py:42]: missing sponsored field

File: src/pycontw2016/urls.py [hunk 2]
```diff
@@ -33,6 +33,7 @@
 urlpatterns += [
     url(r'^ccip/', include('ccip.urls')),
     url(r'^api/sponsors/', include('sponsors.api.urls')),
+    url(r'^api/events/', include('events.api.urls', namespace="events")),
```
Comment (hsuanchi) [inline src/pycontw2016/urls.py:36]: according to api document the url should be like "api/conference/talk/{id}/"

Comment (setmao) [reply src/pycontw2016/urls.py:36]: URL != API path, it means you can find these data in 2020's website with URL.
So I think api/events/ is fine.

File: src/events/api/urls.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from events import api
+
+app_name = 'app'
+
+urlpatterns = [
+    path('talkproposal/<int:pk>', api.TalkDetailAPIView.as_view()),
```
Comment (setmao) [inline src/events/api/urls.py:8]: I prefer use talk/<int:pk>, instead of talkproposal(talkproposal include not accepted talk).

**Review by setmao — CHANGES_REQUESTED** [review]
Follow #1002 
rename src/events/api/talkproposal.py to src/events/api/view.py

File: src/events/api/urls.py (hunk 3 above)
Comment (Maliaotw) [reply src/events/api/urls.py:8]: reasonable

File: src/events/serializers.py (hunk 1 above)
Comment (Maliaotw) [reply src/events/serializers.py:42]: 我在talk_detail.html裡頭找不到"sponsored"是在哪裡定義的，也看不出來是哪個關聯model的值，這個我先todo一下好了，之後再更新。

Comment (mattwang44) [reply src/events/serializers.py]: I assume this list is copied from somewhere else in the repo and the field `sponsored` is provided by [this function](https://github.com/pycontw/pycon.tw/blob/f6948d702f2d6cf1ab70a6352d0b24225b00b024/src/events/templatetags/events.py#L110) for rendering from the Django templates. We still need this field to differentiate if each talk is a sponsored event.

**Review by setmao — APPROVED** [review]
LGTM

**Pull request merged**